### DISCUSSION TITLE
Unresponsive during upload

### DISCUSSION THREAD

**Discussion by pstark556** [photoprism/photoprism#2046] [Q&A] [answered]
During the upload, PhotoPrism keeps freezing. I set it up yesterday and I can upload limited small amounts of photos. ie. 200-500photos. In total I have aprox 10,000 photos I want to backup but if I select all to upload, it will freeze at some point. It's not consistently stopping on the same number. 
 
I have a mysql data base to try to remedy the issue but that made no change. 

The first 30 or so lines "time="2022-02-18T19:00:51-08:00" level=info msg="metadata: failed parsing GPS coordinates in FABAFAA4-EBC6-4512-A813-8262E9C7BC8A.jpeg (exif)"" pops up about three times, but after is a long list of no warnings. Last 5 line say time="2022-02-18T19:00:51-08:00" level=info msg="metadata: failed parsing GPS coordinates in FABAFAA4-EBC6-4512-A813-8262E9C7BC8A.jpeg (exif)"

**Comment by graciousgrey** [comment, marked as answer]
The warnings you mention are not related to the failing upload. 
The web upload (https://docs.photoprism.app/user-guide/library/upload/) is not build to upload 10 000 images.

We recommend using webdav (see https://docs.photoprism.app/user-guide/library/webdav/ and https://docs.photoprism.app/user-guide/sync/webdav/) to move or copy your library to PhotoPrism's originals or import folder. 

This will be much faster.

We should add a limit to the web upload so that there is a proper error/info message instead of a freezing interface.
